2018/Alekseev/FootballLeague/Football_League/ConsoleWorker.cs

```diff
@@ -73,177 +73,231 @@ public static int ChooseDrawingType()
         public static void PrintHorizontalGrid(List<string> horizontalGrid)
```

Comment:
Probably, it's better to move drawers to distinct classes. They have quite many methods processing them.